Models/api.model.js

```diff
@@ -0,0 +1,5 @@
+const endpoint = require("../endpoints.json");
+
+exports.fetchApi = () => {
```

Comment:
As you're not interacting with the database you dont need anything in the model for this ticket.